package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "dependencies": {
+    "jquery": "^2.2",
```

Comment:
Why jquery 2 and not 3 ? 
And why jquery ? Can't we be jquery-free ?

Reply:
When we update the datepicker , if we choose to stay with https://github.com/Eonasdan/tempus-dominus

the version 6 is in beta and does not use jQuery, bootstrap or momentjs. Only depends on popperjs (a very tiny dep)

Reply:
jQuery is not added anymore here directly, it is picked from SonataAdminBundle.